### ISSUE TITLE
Text present under "Get Started" tab visually appears as heading but is not defined as heading programmatically.

### ISSUE THREAD

**Issue by hartra344** [Azure/LogicAppsUX#3639]
User Experience:
Visually challenged users who rely on screen readers using shortcut(H) to understand the structure of the content in the page are getting impacted, as headings are not identified when navigated via heading shortcut(H).

Note: 
User credentials should NOT be included in the bug.
Repro Steps:
Open https://ms.portal.azure.com/ and login via valid credentials.
Navigate to the Left Nav bar and select 'More Services', search 'Integration Accounts' and Select any already created Integration Account
Select Overview in left navigation pane. Navigate all the controls on the blade
Verify if Text present under "Get Started" tab visually appears as heading but is not defined as heading programmatically.
Actual Result:
Text such as "Build, deploy, and scale web apps on a fully managed platform", "Link to Logic App", "Create an XML schema" and "Use artifacts in logic apps" present under "Get Started" tab visually appears as heading but is not defined as heading programmatically.
Screen reader behavior: While navigating in heading mode after "ux-test-integration-account" text screen reader announces as "No next heading".
Expected Result:

Text such as "Build, deploy, and scale web apps on a fully managed platform", "Link to Logic App", "Create an XML schema" and "Use artifacts in logic apps" present under "Get Started" tab should be defined as heading programmatically i.e "Build, deploy, and scale web apps on a fully managed platform" text can be defined as <H2> and remaining text should be <H3>.
ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance/previous test pass bug id's:
NA
UIA/DOM details available:
Yes
MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://microsoft.sharepoint.com/:w:/s/accessibility/EXdDUZzldv5MgwDaMHnTzcMBcT8opKGncYjU-gCJ4uqQdg?e=9nuVmt)
Reference Links
[Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

https://msazure.visualstudio.com/One/_workitems/edit/24783093
